### PR TITLE
Update element pan to pass drag functions to children

### PR DESCRIPTION
**Pull request by cybadave**

### CODE REVIEW

File: components/elementPan/ElementPan.js [hunk 1]
```diff
@@ -492,6 +492,11 @@ export default class ElementPan extends React.PureComponent {
             onWheel
         } = this
         const { poof, x, y } = this.state
+        const childrenWithCallbacks = React.Children.map(this.props.children, child => React.cloneElement(child, {
+            onDragStart: this.onDragStart,
+            onDragMove: this.onDragMove,
+            onDragStop: this.onDragStop
+        }));
```
Comment (diogobeda) [inline components/elementPan/ElementPan.js]: that sweet composition ❤️

Comment (jimmed) [reply components/elementPan/ElementPan.js]: awwwwwwwwwwyeeeeeeeeeeah

Comment (victorhtc) [reply components/elementPan/ElementPan.js]: Is it really needed? It makes the children elements too coupled to ElementPan.

Comment (victorhtc) [inline components/elementPan/ElementPan.js]: Is it really needed? It makes the children elements too coupled to ElementPan. These prototype methods are exposed by the element ref, maybe injecting them in the children props would be a better option?

Comment (cybadave) [reply components/elementPan/ElementPan.js]: Unfortunately, yes: The idea is that a child object which implements react-draggable core can defer default behaviour to the parent ElementPan's implementation. ( Vertically draggable project roles which can still pan the page on left or right drag )

**Pull request merged**